src/client/Item/Items.ts

```diff
@@ -72,6 +81,72 @@ export class Items {
     return new QueryIterator(this.clientContext, query, options, fetchFunction, this.container.url);
   }
 
+  /**
+   * Create a `ChangeFeedIterator` to iterate over pages of changes
+   * @param partitionKey
+   * @param changeFeedOptions
+   */
+  public readChangeFeed(
```

Comment:
usage example would be good to have. We generate jsdoc from this, right?

Reply:
I can add a code sample here, yeah. It will show up in our reference docs.

Reply:
Yes, please, would make big difference.